test/Microsoft.AspNetCore.Server.Kestrel.Core.Tests/FrameRequestHeadersTests.cs

```diff
@@ -30,7 +30,6 @@ public void SettingUnknownHeadersWorks()
 
             headers["custom"] = new[] { "value" };
 
-            Assert.NotNull(headers["custom"]);
```

Comment:
FYI, this returned `StringValues` which is a value type (aka can never be null).